README.md

```diff
@@ -13,6 +13,64 @@
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
 
+## Migration to v1
+
+> [!WARNING]
+>
+> Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.
```

Comment:
> Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.
 
I would split that line, so we can see the part about backuping stuff better.
 
  > Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file. 
  > If you are using a local state file, you can simply make a copy of your terraform.tfstate file in your project directory.